src/main/java/com/google/sps/servlets/MealServlet.java

```diff
@@ -33,21 +44,47 @@
 @WebServlet("/meal/*")
 public class MealServlet extends HttpServlet {
 
-    private final HashMap<Long, Meal> dishes = new HashMap<Long, Meal> ();
-
     @Override
     public void init() {
         // TODO(sandatsian): implement uploading of dataset from website here
-        dishes.put(0L, new Meal(0L, "Fried potato", "Fried potato with mushrooms and onion.",
-            new ArrayList<>(Arrays.asList("Potato", "Onion", "Mushrooms", "Oil")), "main"));
-        dishes.put(1L, new Meal(1L, "Italian pizza", "Pizza with pineaple, sausage and tomato.",
-            new ArrayList<>(Arrays.asList("Flour", "Water", "Sausage", "Tomato", "Pineaple", "Cheese")), "pizza"));
-        dishes.put(2L, new Meal(2L, "Vegetable soup", "Vegetable soup with onion.",
-            new ArrayList<>(Arrays.asList("Potato", "Onion", "Cabbage", "Mushrooms", "Water", "Carrot", "Pumpkin")), "soup"));
-        dishes.put(3L, new Meal(3L, "Chocolate cake", "Chocolate cake with butter cream and strawberry.",
-            new ArrayList<>(Arrays.asList("Flour", "Water", "Butter", "Cacao", "Chocolate", "Sugar", "Strawberry", "Eggs")), "dessert"));
-        dishes.put(4L, new Meal(4L, "Hot chocolate", "Hot chocolate with sugar and caramel.",
-            new ArrayList<>(Arrays.asList("Cacao", "Sugar", "Milk", "Caramel", "Vanil")), "drinks"));
+        DatastoreService datastore = DatastoreServiceFactory.getDatastoreService(); 
```

Comment:
I'm wondering whether this code will keep on adding new items whenever we start the server. I'm not sure how this class works - does it create a new DataStore every time we use it, or does it connect to the same instance somewhere?